### ISSUE TITLE
[Bug] REPO_INDEXER doesn't index mirrored repository

### ISSUE THREAD

**Issue by ghost** [go-gitea/gitea#11200]
- Gitea version (or commit ref): 1.11.4
- Git version: Not relevant
- Operating system: docker image
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist: Not relevant

## Description

The REPO_INDEXER doesn't index the mirrored repository automatically. I have to remove the `.bleve` every time to start a new reindex for all repositories.

It would be great if the indexing can be  triggered manually on a selected repository. Would be better if indexing is automatically  triggered after pulling from remotes. 

## Screenshots
Here is my indexer configuration
```
[indexer]
ISSUE_INDEXER_PATH = /data/gitea/indexers/issues.bleve
REPO_INDEXER_ENABLED = true
REPO_INDEXER_PATH = /data/gitea/indexers/repos.bleve
UPDATE_BUFFER_LEN = 20
MAX_FILE_SIZE = 1048576
REPO_INDEXER_INCLUDE = 
REPO_INDEXER_EXCLUDE = resources/bin/**
```

**Comment by simon-on-gh** [comment]
Is it possible to call ``NotifyPushCommits`` manually form GUI, or via some command-line?
https://github.com/go-gitea/gitea/blob/310699bca71f15cc7b46363800574ba6e74d8c5c/modules/notification/indexer/indexer.go#L126 

Currently it is only called from
https://github.com/go-gitea/gitea/blob/6034f8bcaaa5348fee775d2307ff03162130a088/modules/repofiles/action.go#L257

**Comment by simon-on-gh** [comment]
For mirrored commits, there is a ``NotifySyncPushCommits``
https://github.com/go-gitea/gitea/blob/2677d071f911fb91f382acfedaedc251bd807f70/services/mirror/mirror.go#L412
But it doesn't call the indexer like ``NotifyPushCommits``
https://github.com/go-gitea/gitea/blob/310699bca71f15cc7b46363800574ba6e74d8c5c/modules/notification/indexer/indexer.go#L126

**Comment by zeripath** [comment]
To me it seems that simply copying the code for NotifyPushCommits would suffice to fix this issue.

**Comment by simon-on-gh** [comment]
It seems that this bug wasn't fixed in the 1.11.5 release. Was the backport included in this release?

**Comment by zeripath** [comment]
https://github.com/go-gitea/gitea/blob/83f8414e1ee769a62ae813b10f602a800eb76ac5/modules/notification/indexer/indexer.go#L127
